sdk/build.gradle

```diff
@@ -30,3 +30,10 @@ animalsniffer {
             sourceSets.main
     ]
 }
+
+processResources {
+    new File(
+            projectDir.getAbsolutePath() + File.separator + ["src", "main", "resources"].join(File.separator),
+            "version.properties"
```

Comment:
I think we should put this into a package, e.g. io/opentelemetry/sdk/version.properties